ui/package.json

```diff
@@ -46,21 +47,21 @@
     "ngx-pagination": "3.1.1",
     "ngx-toastr": "8.5.0",
     "sockjs-client": "1.1.4",
-    "web-animations-js": "2.3.1",
+    "web-animations-js": "github:angular/web-animations-js#release_pr208",
```

Comment:
Well this dependency is smelling a lot. Shouldn't we rely on what's in npm repos?